### DISCUSSION TITLE
Is it possible to put array of TableRow object in a Table object

### DISCUSSION THREAD

**Discussion by acarvajal12** [dolanmiu/docx#1442] [Q&A] [answered]
Hello guys,

Good day!

I've seen a snippet in Usage document on how to create a table using array of TableRow:
const table = new Table({
    rows: [Array of `TableRow`s]
});

But unfortunately it flags an error.
![Screenshot](https://user-images.githubusercontent.com/100741845/156491722-4e0d3ae6-e4f4-4ae2-b5a3-ddcd3cc51cd4.png)

Below is the code snippets how I implemented:
      var rows: TableRow[] = [];
      
      var headerRow = new TableRow({
        children:[],
        tableHeader: true,
      });
      var ctr = 0;
      for (const col of headers) {
        headerRow.addCellToIndex(
          new TableCell({
            children: [new Paragraph({
              text: col,
              heading: HeadingLevel.HEADING_1,
              alignment: AlignmentType.CENTER,})
            ],
        }), ctr
        );
        ctr++;
      }
      rows.push(headerRow);

      for (const value of datas) {
        const row = new TableRow({children:[],});
        ctr = 0;
        for (const key of keys) {
          var txt = isNil(value[key]) ? '' : value[key];
          row.addCellToIndex(
            new TableCell({
              children: [new Paragraph({
                text: txt,
                alignment: AlignmentType.CENTER,})
              ],
            }), ctr
          );
          ctr++;
        }
      rows.push(row);
      }

      //Is this the correct way how to add an array of TableRow in a Table object?
      const table = new Table({
        rows: [Array of rows,]
      });

Your answer is highly appreciated.
Thank you.

**Comment by dolanmiu** [comment, marked as answer]
Try doing this:

```ts
const table = new Table({
  rows: rows
});
```

or:

```ts
const table = new Table({
    rows: [...rows]
  });
```

**Reply by acarvajal12** [reply]
Okay, thank you.
